addons/website/static/src/snippets/s_dynamic_snippet/000.js

```diff
@@ -159,6 +159,7 @@ const DynamicSnippet = publicWidget.Widget.extend({
     /**
      * Method to be overridden in child components in order to prepare QWeb
      * options.
+     *
```

Comment:
Detail but would have been better in the next commit :o (or not at all)

Reply:
The next commit is not about the `_getQWebRenderOptions`, it would have been worse there :shrug: So it could have been on its own commit maybe but was not feeling like fixing the whole file :grimacing:  And it's hardly noise in this commit and does not even break code history as it is a new line (but I would happily have broken history for a linter/convention error related to the code I fix anyway)